src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,46 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        List<Book> books = List.of(
+                new Book(null, "Book1", new BigDecimal(100)),
+                new Book(null, "Book2", new BigDecimal(150)),
+                new Book(null, "Book3", new BigDecimal(50))
```

Comment:
Creating `Book` objects with `null` id is a bit misleading since it's not immediately clear why the id is set to `null`. If the `id` is meant to be generated by the database, then it would be better to have a constructor that does not require an `id` parameter at all, which would make the intent clearer.